src/load/azext_load/data_plane/utils/constants.py

```diff
@@ -33,3 +33,10 @@ class LoadTestConfigKeys:
 class HighScaleThreshold:
     MAX_ENGINE_INSTANCES_PER_TEST_RUN = 45
     MAX_DURATION_HOURS_PER_TEST_RUN = 3
+
+
+@dataclass
+class LoadCommandsConstants:
+    CONVERT_TO_JMX_CONFIRM_PROMPT = "Once the test is converted, you will not be able to revert. " \
+        "Please ensure you understand the implications.\n" \
```

Comment:
Also get it reviewed by PM

Reply:
I took it from portal, @saloniagrawal1997 to review

Reply:
Once the test is converted, the process cannot be reversed.

This much is fine.

Reply:
We cn remove the line on implications